en/installation.rst

```diff
@@ -34,7 +34,8 @@ Requirements
 While a database engine isn't required, we imagine that most applications will
 utilize one. CakePHP supports a variety of database storage engines:
 
--  MySQL (5.1.10 or greater)
+-  MySQL (5.5.3 or greater)
```

Comment:
What about 

     (5.1.10 or greater, 5.5.3+ for full utf8mb4 support) 

?

Reply:
The driver really needs to change to utf8mb4 and then will be broken by default unless configured differently. People can still try to run things on older platforms but we should not encourage nor say that we officially support them IMHO. Also we don't run tests on 5.1, do we?

I know it is not the same but, to set some perspective:

- MySQL 5.1 2008, ~9 years ago (~PHP 5.2)
- MySQL 5.5 2010, ~7 years ago
- MySQL 5.6 2014, ~3 years ago (~PHP 5.6)
- MySQL 5.7 2015, ~2 years ago (~PHP 7.0)

Reply:
I am ok with suggesting version of mysql that is newer. Beyond utf8 support there are many advantages to using newer Mysql, like online schema changes and json data types.